cmd/dashboard/command_history.js

```diff
@@ -24,6 +29,19 @@ class CommandHistory {
     this.pointer++;
     return this.history[this.pointer];
   }
-}
 
+  loadHistory() {
+    if (fs.existsSync(this.cmdHistoryFile)) {
+      fs.readFileSync(this.cmdHistoryFile)
+        .toString()
+        .split('\n')
+        .forEach((cmd) => { this.addCommand(cmd); })
+    }
+  }
+
+  saveToHistoryFile(cmd) {
+    fs.appendFileSync(this.cmdHistoryFile, cmd + "\n");
```

Comment:
I think we should limit the number of commands we store here, as it could grow to infinity and also, because getting more than 20 commands in history is useless.